### PR TITLE
Update MU_BASECORE pin for new Openssl version

### PR DESCRIPTION
**Pull request by kenlautner**
## Description

Updated the MU_BASECORE submodule and updated the repo to accommodate the changes required for that.

For each item, place an "x" in between `[` and `]` if true. Example: `[x]`.
_(you can also check items in the GitHub UI)_

- [ ] Impacts functionality?
  - **Functionality** - Does the change ultimately impact how firmware functions?
  - Examples: Add a new library, publish a new PPI, update an algorithm, ...
- [x] Impacts security?
  - **Security** - Does the change have a direct security impact on an application,
    flow, or firmware?
  - Examples: Crypto algorithm change, buffer overflow fix, parameter
    validation improvement, ...
- [ ] Breaking change?
  - **Breaking change** - Will anyone consuming this change experience a break
    in build or boot behavior?
  - Examples: Add a new library class, move a module to a different repo, call
    a function in a new library class in a pre-existing module, ...
- [ ] Includes tests?
  - **Tests** - Does the change include any explicit test code?
  - Examples: Unit tests, integration tests, robot tests, ...
- [ ] Includes documentation?
  - **Documentation** - Does the change contain explicit documentation additions
    outside direct code modifications (and comments)?
  - Examples: Update readme file, add feature readme file, link to documentation
    on an a separate Web page, ...

## How This Was Tested

Tested with the shell app and tested ECC crypto separately.

## Integration Instructions

A new official binary will be built after this goes in.  That will be updated on the MU_BASECORE side so you'll need to take that.

### CODE REVIEW

**Comment by apop5** [comment]
PR title needs updated.

File: .azuredevops/pipelines/ReleaseBuild.yml [hunk 1]
```diff
@@ -333,22 +333,25 @@ stages:
           script: >
             mkdir $(Build.StagingDirectory)/NupackOutput;
             $configFilePath = "$(Pipeline.Workspace)/Extras/edk2-BaseCryptoDriver.config.json";
+            $licenseFile = "$(Pipeline.Workspace)/Extras/License.txt";
```
Comment (apop5) [inline .azuredevops/pipelines/ReleaseBuild.yml:336]: is the ; necessary?

File: .azuredevops/pipelines/ReleaseBuild.yml [hunk 2]
```diff
@@ -333,22 +333,25 @@ stages:
...
 
             Write-Host nuget-publish
             --Operation Pack
             --OutputLog "$(Build.StagingDirectory)/NugetPackagingLog.txt"
             --ConfigFilePath "$configFilePath"
             --InputFolderPath "$(Pipeline.Workspace)/Package"
             --Version "$(publish_version)"
-            --OutputFolderPath "$(Build.StagingDirectory)/NupackOutput";
+            --OutputFolderPath "$(Build.StagingDirectory)/NupackOutput"
+            --CustomLicensePath "$licenseFile";
```
Comment (apop5) [inline .azuredevops/pipelines/ReleaseBuild.yml:345]: is the ; necessary?

**Review by apop5 — APPROVED** [review]

**Review by spbrogan — APPROVED** [review]

File: .azuredevops/pipelines/ReleaseBuild.yml (hunk 2 above)
Comment (kenlautner) [reply .azuredevops/pipelines/ReleaseBuild.yml:345]: Yes.  It's required to mark the ending of the write-host command.

**Pull request merged**